packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -498,7 +514,7 @@ export class RunningSummarizer implements IDisposable {
 				};
 
 				if (delaySeconds > 0) {
```

Comment:
Can you please move this code dealing with delays to the end of the loop?
With intention that even if we bail out after 2 attempts (or even 1), we would respect retryAfter, i.e. wait whatever service told us to wait before repeating or closing summarizer.
In other words, it does not matter if we do another iteration or exit and reboot summarizer, we need to ensure that delay is respected before we attempt to post another summary.